__tests__/app.test.js

```diff
@@ -144,3 +144,78 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+// POST
+describe("POST /api/articles/:article_id/comments", () => {
+  test("status: 201 - responds with created comment", () => {
+    const newComment = {
+      username: "butter_bridge",
+      body: "This is a great article",
+    };
+    return request(app)
+      .post("/api/articles/7/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body: { comment } }) => {
```

Comment:
Please check comment_id here to see if this is the new comment or just another with the same content.